File: templates/py/benchmarks/start-up-state.py
import time
import json
import sys
# %imports
def runner(*args):
    output=0
    # %vars
    # %pre
    start = time.time()
    # %content
    end = time.time()
    # %post
    res = {
        "input": args,
        "output": output,
        "time": end - start
    }
    json.dump(res, sys.stdout)

File: templates/js/benchmarks/start-up-state.js
// %imports

async function runner(...args){
    let output = 0;
    // %init
	// %gc
    // %pre
    let start = performance.now();
    // %content
    let diff = performance.now();
    // %post
	// %gc
    let result = {
        input:JSON.stringify((args)),
        output:output,
        time: (diff - start)/1000
    };
    console.log(JSON.stringify(result));
}

module.exports = {runner};
// %runner_call